### PR TITLE
GetClaimables call and example, readme adjustment

### PR DESCRIPTION
**Pull request by Chriserus**

### CODE REVIEW

File: src/apis/platformvm/interfaces.ts [hunk 1]
```diff
@@ -66,6 +66,13 @@ export interface GetCurrentValidatorsParams {
   nodeIDs?: string[]
 }
 
+export interface GetClaimablesParams {
+  addresses: string[]
+  depositTxIDs: string[]
+  locktime: BN
```
Comment (c4t-ag) [inline src/apis/platformvm/interfaces.ts]: If this interface is request params, we don't use BN anymore -> pls. convert use string

File: src/apis/platformvm/api.ts [hunk 2]
```diff
@@ -528,6 +529,34 @@ export class PlatformVMAPI extends JRPCAPI {
...
+      locktime,
+      threshold,
+      addresses,
+      depositTxIDs
+    }
+    const response: RequestResponseData = await this.callMethod(
+      "platform.getClaimables",
+      params
+    )
+    return response.data.result
```
Comment (c4t-ag) [inline src/apis/platformvm/api.ts]: What response is provided? We usually have structs / interface defined in interfaces.ts if the return result is more than a simple string or number.

**Review by c4t-ag — CHANGES_REQUESTED** [review]

File: examples/platformvm/getClaimables.ts [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+const main = async (): Promise<any> => {
+  await InitAvalanche()
+
+  const txIDs: string[] = ["2nmH8LithVbdjaXsxVQCQfXtzN9hBbmebrsaEYnLM9T32Uy2Y4"]
+  const currentValidators: object = await pchain.getClaimables(
+    pAddressStrings,
+    txIDs
+  )
+  console.log(currentValidators)
```
Comment (c4t-ag) [inline examples/platformvm/getClaimables.ts]: currentValidators

File: src/apis/platformvm/api.ts [hunk 4]
```diff
@@ -528,6 +530,36 @@ export class PlatformVMAPI extends JRPCAPI {
...
+      depositTxIDs
+    }
+    if (typeof locktime !== "undefined") {
+      params.locktime = locktime
+    }
+    const response: RequestResponseData = await this.callMethod(
+      "platform.getClaimables",
+      params
+    )
+    return response.data.result
```
Comment (c4t-ag) [inline src/apis/platformvm/api.ts]: Are you sure that response.data.result is properly deserialized into the structure?

**Review by c4t-ag — CHANGES_REQUESTED** [review]

File: examples/platformvm/getClaimables.ts (hunk 3 above)
Comment (Chriserus) [reply examples/platformvm/getClaimables.ts]: copy paste mistake! correcting

File: src/apis/platformvm/api.ts (hunk 4 above)
Comment (Chriserus) [reply src/apis/platformvm/api.ts]: FYI, we found a bug with Evgenii and this PR will wait for the fix from his side. I will update you as soon as it's fixed in node's code

File: examples/platformvm/getClaimables.ts [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+let pKeychain: KeyChain
+let pAddresses: Buffer[]
+let pAddressStrings: string[]
+
+const InitAvalanche = async () => {
+  await avalanche.fetchNetworkSettings()
+  pchain = avalanche.PChain()
+  pKeychain = pchain.keyChain()
+  pKeychain.importKey(privKey)
+  pAddresses = pchain.keyChain().getAddresses()
```
Comment (c4t-ag) [inline examples/platformvm/getClaimables.ts]: Unused

File: src/apis/platformvm/api.ts [hunk 6]
```diff
@@ -528,6 +530,41 @@ export class PlatformVMAPI extends JRPCAPI {
...
+    const response: RequestResponseData = await this.callMethod(
+      "platform.getClaimables",
+      params
+    )
+    const result = response.data.result
+    return {
+      depositRewards: new BN(result.depositRewards),
+      validatorRewards: new BN(result.validatorRewards),
+      expiredDepositRewards: new BN(result.expiredDepositRewards)
+    }
```
Comment (c4t-ag) [inline src/apis/platformvm/api.ts]: as GetClaimablesResponse

**Review by c4t-ag — CHANGES_REQUESTED** [review]

**Review by c4t-ag — APPROVED** [review]

**Pull request merged**